navigation-material/src/androidTest/java/com/google/accompanist/navigation.material/BottomSheetNavigatorTest.kt

```diff
@@ -368,6 +388,379 @@ internal class BottomSheetNavigatorTest {
         }
     }
 
+    @Test
+    fun testSheetContentSizeChangeDuringAnimation_opensSheet_shortSheetToShortSheet() {
+        val animationDuration = 2000
+        val animationSpec = tween<Float>(animationDuration)
+        lateinit var navigator: BottomSheetNavigator
+        lateinit var navController: NavHostController
+        var height: Dp by mutableStateOf(20.dp)
+
+        composeTestRule.setContent {
+            navigator = rememberBottomSheetNavigator(animationSpec)
+            navController = rememberNavController(navigator)
+            navController.currentBackStackEntryAsState().value
```

Comment:
Why do we need to read the value here? This should work whether or not we add extra recompositions, right?

Reply:
Yep, good catch!

Reply:
Yep, not needed. Done